scripts/deployment20/deploy.ts

```diff
@@ -446,7 +458,7 @@ export class DeploymentSystem extends DeployedSystemHelpers {
     !useInch &&
       addLocalEntries &&
       (await deploySwapContract.setPool(
-        this.config.common.STETH.address,
+        this.config.common.WSTETH.address,
```

Comment:
```suggestion
        this.config.common.STETH.address,
```

Changing that will possibly mess with mainnet uniswap tests. Any STETH positions

Reply:
This should be reverts as I don't use setupLocalSystem at all